app/scripts/lib/user-agent.js

```diff
@@ -31,6 +31,15 @@ define(function (require, exports, module) {
       },
 
       /**
+       * Check if on a desktop device
+       *
+       * @returns {Boolean}
+       */
+      isDesktop () {
+        return ! this.isAndroid() && ! this.isIos();
```

Comment:
We could flip my silly suggestion round so it's a bit less silly:

```js
isPlatformWithAppStore () {
   return this.isAndroid() || this.isIos();
}
```

Warmer?